controllers/chat/group/leaveGroupMember.js

```diff
@@ -16,6 +23,28 @@ const leaveGroupMember = async (req, res) => {
       const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
         await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
 
+      const client = new StreamChat(
+        Environment.GETSTREAM_API_KEY,
+        Environment.GETSTREAM_API_SECRET
+      );
+      const currentChannel = await client.channel(CHANNEL_TYPE_STRING.GROUP, channelId);
+
+      const textOwnUser = `You leave from this group`;
+      const textTargetUser = `${ownUser.username} has been leave from this group`;
+      const textDefaultUser = `${ownUser.username} has been leave from this group`;
+      const members = betterChannelMember.map((member) => member.user_id);
+
+      await currentChannel.sendMessage({
+        text: textDefaultUser,
+        own_text: textOwnUser,
+        other_text: textTargetUser,
+        type: 'system',
+        user_id: userId,
+        system_user: userId,
+        isSystem: true,
+        members: members
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The system message being sent when a user leaves a group is hardcoded in English. This may not be suitable for applications with international users. Consider using a localization library to support multiple languages.

Also, it seems like `textOwnUser`, `textTargetUser`, and `textDefaultUser` are all set to the same value. If they are intended to be different, this should be corrected. If they are always the same, consider simplifying this to a single variable.

Lastly, the `user_id` and `system_user` fields are both set to `userId`. If these fields are meant to represent different entities (i.e., the user who triggered the action vs. a system user), they should not be the same. If they are always the same, consider removing one to reduce redundancy.

```diff
- const textOwnUser = `You leave from this group`;
- const textTargetUser = `${ownUser.username} has been leave from this group`;
- const textDefaultUser = `${ownUser.username} has been leave from this group`;
+ const textMessage = `${ownUser.username} has left the group`;

- await currentChannel.sendMessage({
-   text: textDefaultUser,
-   own_text: textOwnUser,
-   other_text: textTargetUser,
-   type: 'system',
-   user_id: userId,
-   system_user: userId,
-   isSystem: true,
-   members: members
- });
+ await currentChannel.sendMessage({
+   text: textMessage,
+   type: 'system',
+   user_id: userId,
+   isSystem: true,
+   members: members
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->